fastlane/metadata/android/hi-IN/title.txt

```diff
@@ -1 +1 @@
-कैटीमा — वफादारी कार्ड वॉलेट
+कैटीमा — Loyalty कार्ड वॉलेट
```

Comment:
@Govindgopalyadav I don't think "loyalty" is a Hindi word? A "loyalty card" is a discount card in a store, it's a very English term: https://en.wikipedia.org/wiki/Loyalty_program?wprov=sfla1